frontend/src/components/organisms/ManageAttendees.tsx

```diff
@@ -364,26 +398,34 @@ const ManageAttendees = ({}: ManageAttendeesProps) => {
       // TODO: Double check endpoint
       // It currently returns list of ALL users, not just attendees for a specific event
       const { data } = await api.get(
-        `/users?eventid=${eventid}&limit=${paginationModel.pageSize}`
+        `/events/${eventid}`
```

Comment:
you don't want to change this. There was a slight bug with it but I have pushed a change now. the correct query will be `await api.get(`/users?eventId=${eventid}&limit=${paginationModel.pageSize}`);`. You can log this in the console to see that it gives you all the information you need for a user's enrollment. No need to make another endpoint to get the user's enrollment status